cmake/TorchConfig.cmake.in

```diff
@@ -71,8 +79,6 @@ if (@USE_CUDA@)
       ${LIBNVTOOLSEXT}
       ${CUDA_LIBRARIES})
   endif()
-  find_library(C10_CUDA_LIBRARY c10_cuda PATHS "${TORCH_INSTALL_PREFIX}/lib")
```

Comment:
Are these two lines removed intentionally?
I'm not familiar with this part but with/without it doesn't seem to affect Android build according to my local test...

Reply:
Uh, sorry! Great catch. I think this went wrong during rebasing and the line should stay (based on the hypothesis that `@USE_CUDA@` doesn't apply to android. (As is apparent from the above, we need to avoid the build of android custom operator libraries to search in the system (i.e. non-android) locations. But this line should stay.)